index.html

```diff
@@ -390,29 +394,36 @@
   insufficient implementation experience (reports).
   </p>
   <ul>
-  <li>Whole features described in <a href="#apikeysecurityscheme"></a>, <a href="#certsecurityscheme"></a>, 
+  <li>Whole sections related to the security schemes described in <a href="#apikeysecurityscheme"></a>, <a href="#certsecurityscheme"></a>, 
```

Comment:
Wouldn't it be better to call these security schemes features?
I think the usual terminology is "features at risk", no?

Reply:
`<li>Security schemes <a href="...">APIKeySecurityScheme</a>, CertSecurityScheme, PSKSecurityScheme, PublicSecurityScheme, PoPSecurityScheme, and OAuth2SecurityScheme</li>`
?